.github/workflows/lint.yaml

```
@@ -15,11 +15,11 @@ jobs:
       steps:
         - name: Checkout code
           uses: actions/checkout@v4 
-          with:
-              python-version: '3.11'
   
         - name: Set up Python
           uses: actions/setup-python@v4  
+          with:
+              python-version: '3.13'
```

```suggestion
          with:
              python-version: '3.x'
```

This chooses the latest version of python3